dev/Generated/NumberBox.properties.h

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// DO NOT EDIT! This file was generated by CustomTasks.DependencyPropertyCodeGen
+#pragma once
+
+class NumberBoxProperties
+{
+public:
+    NumberBoxProperties();
+
+    void BasicValidationMode(winrt::NumberBoxBasicValidationMode const& value);
+    winrt::NumberBoxBasicValidationMode BasicValidationMode();
+
+    void FractionDigits(int value);
+    int FractionDigits();
+
+    void Header(winrt::hstring const& value);
+    winrt::hstring Header();
+
+    void HyperScrollEnabled(bool value);
+    bool HyperScrollEnabled();
+
+    void IncrementPrecision(double value);
+    double IncrementPrecision();
+
+    void IntegerDigits(int value);
+    int IntegerDigits();
+
+    void IsDecimalPointAlwaysDisplayed(bool value);
+    bool IsDecimalPointAlwaysDisplayed();
+
+    void IsZeroSigned(bool value);
+    bool IsZeroSigned();
+
+    void MaxValue(double value);
+    double MaxValue();
+
+    void MinMaxMode(winrt::NumberBoxMinMaxMode const& value);
+    winrt::NumberBoxMinMaxMode MinMaxMode();
+
+    void MinValue(double value);
+    double MinValue();
+
+    void NumberRounder(winrt::NumberBoxNumberRounder const& value);
+    winrt::NumberBoxNumberRounder NumberRounder();
```

Comment:
So many properties, and still not enough to show for example sign of degree or other metric suffix after the text. Is there a possibility to provide custom value converter between string and double?

Reply:
Resolving since this was taken over to the proposal thread

Reply:
> So many properties, and still not enough to show for example sign of degree or other metric suffix after the text. Is there a possibility to provide custom value converter between string and double?

I have a proposal that is going to be pitched, which will handle Suffix and Prefix properties to all TextBoxes, which NumberBox could adopt also. #784